pg_diffix.vcxproj

```diff
@@ -78,13 +78,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>

```

Comment:
ah, this! I thought this indicates the exact version not "equal or greater than". That is, it might not work with 2022 now (I think I've put in 141 mistakenly, and VS 2019 didn't work). Please, double check 2022.

Reply:
I did a test and it still works, but I have both build tools versions installed. Not sure if it will still work if I uninstall the 2019 version of the tools, but I don't want to try it out of fear of messing up my build environment. The project toolset can be upgraded back easily enough though.

Reply:
From what I know, node.js installs the 2019 build tools, so it is better to depend on those for easier building Dashboards on GitHub.